aptos-move/aptos-release-builder/src/validate.rs

```diff
@@ -216,7 +216,7 @@ impl NetworkConfig {
                 "0x1::aptos_governance::GovernanceEvents",
                 "create_proposal_events",
                 None,
-                Some(100),
+                Some(1),
```

Comment:
Do we eventually need to handle pagination here?

Reply:
I don't think so. We should only need one event here cuz we just created one. One thing to handle tho is if we need any cleanups after some txn failed in the middle.